src/commands/set.js

```diff
@@ -98,9 +99,30 @@ const setString = (dotNotation, value) => {
 	return attemptWriteToFile(value, dotNotation);
 };
 
+const setNethash = (dotNotation, value) => {
+	if (
+		dotNotation === 'api.network' &&
+		!Object.keys(NETHASHES).includes(value)
+	) {
+		if (value.length !== 64) {
+			throw new ValidationError(
+				'Value must be a hex string with 64 characters.',
```

Comment:
Actually we should inform users they can use `main`, `test` and `beta` here.

Reply:
how about changing error to
`Value must be a hex string with 64 characters. Alternatively, "main", "test" and "beta" can be used`